### ISSUE TITLE
Add caire image resizer as sample function.

### ISSUE THREAD

**Issue by alexellis** [openfaas/store#36]
I think we should add an image resizer into the store.

This one looks to have shot up to popularity with 8k stars: https://github.com/esimov/caire

It does smart things like scale/resize images while retaining the scale for faces, it can even extend the width of images etc.

Not sure of the best user interface, we could use a string like a human-readable sentence - or JSON as the input. JSON is obviously easier. Human-readable string may be: "Resize this URL https:// to 100 pixel width"

**Comment by martindekov** [comment]
It seems interesting, can I take a shot at this?

**Comment by alexellis** [comment]
Perhaps @esimov could take a look at it? My only concern is whether the resizing would time-out.

**Comment by alexellis** [comment]
Yes we can configure it. Most return in under 10s though. Enjoy your break

**Comment by esimov** [comment]
@alexellis i would opt for a json as an input source. What do you think?

**Comment by alexellis** [comment]
I think that might be a good choice. The users would just have to know how to input the data. Maybe we should extend the UI to create samples/examples of inputs?

**Comment by esimov** [comment]
Hey @alexellis i have integrated Caire as an OpenFaaS function. Here is the PR https://github.com/openfaas/store/pull/41. Please check it and let me know if something is missing or needs to be corrected.

**Comment by psyhomb** [comment]
https://github.com/openfaas/store/issues/74
